### PR TITLE
Mission control center component

### PR DESCRIPTION
**Pull request by Schrese**
# Description
This feature adds the "Mission Control" component as per the Figma styling. Later, we will need to change the checkboxes to be two images that swap (because the child will not be 'checking' them, they will be getting checked off from our end once their tasks are completed), but for now, that isn't a part of our release. 

Fixes #

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Change Status

- [x] Complete, tested, ready to review and merge
- [ ] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

## Has This Been Tested

- [x] Yes
- [ ] No
- [ ] Not necessary

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] My code has been reviewed by at least one peer
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] There are no merge conflicts

### CODE REVIEW

File: src/components/pages/MissionControl/RenderMissionControl.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+import draw_icon from '../../../assets/icons/draw_icon.svg';
+import read_icon from '../../../assets/icons/read_icon.svg';
+import write_icon from '../../../assets/icons/write_icon.svg';
+
+import '../../../styles/MissionControl.less';
+const RenderMissionControl = props => {
+  // function onChange(e) {
+  //   console.log(`checked=${e.target.checked}`);
+  // }
+
```
Comment (Nick-Ohman) [inline src/components/pages/MissionControl/RenderMissionControl.js:14]: just deleted these lines out and everything else looks good!

**Review by Nick-Ohman — COMMENTED** [review]
just those lies and were good

**Review by bramirez96 — DISMISSED** [review]
Thanks for fixing the index! Should let the app build on production

**Review by bramirez96 — APPROVED** [review]
Looks great!

**Review by Nick-Ohman — APPROVED** [review]
Looks great!

**Review by SandraCoburn — APPROVED** [review]
Really specific class styles in your less file. Good job!

**Pull request merged**